pyproject.toml

```
@@ -1,21 +1,43 @@
 [tool.pytest.ini_options]
+testpaths = ["tests"]
 minversion = "6.0"
-xfail_strict=true
-addopts = [
-    "--color=yes",
-]
+xfail_strict = true
+addopts = ["--color=yes"]
 
-[tool.black]
+[tool.ruff]
 line-length = 100
 
```

This should be `"py39"`, but I don't think we want to tackle that in this PR.

```suggestion
target-version = "py37"

```